src/main/java/tecna/TaskList.java

```diff
@@ -0,0 +1,103 @@
+package tecna;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> tasks;
+    private int size;
```

Comment:
Perhaps you could use `tasks.size()` instead of maintaining a separate variable?